packages/ui/src/ui/Toast/styles.ts

```diff
@@ -104,18 +96,24 @@ export const ToastContainerStyle = createGlobalStyle`
   background: #f1c40f;
 }
 .Toastify__toast--error {
-  background: #152935;
-  color: #ffffff
+  background: #e74c3c;
```

Comment:
Wondering why change from the previous dark gray background? Mainly since the colors already aligned with the latest PUI.

You can see here https://polymathnetwork.github.io/Polymesh-UI/?path=/story/snackbar--examples PUI just uses different icons and icon background colors to differentiate the types of messages.

Since this was already in line with that (dark background with yellow icon — alert type), wondering why it needed to be further modified?

Reply:
@wahidrahim well, no deliberate changes were made to these colors. The toast close button was not working due to a css issue, and I had to pull latest styles from react toastify, which obviously changed colors.

It's worth noting that we're using React Toastify in an unorthodox way. Per its docs, we need to import their styles via `import 'react-toastify/dist/ReactToastify.css';`, but that never worked, and hence we had to copy over those styles to our repo.

In any case, I rolled that `background` color to the dark one we had earlier.

Reply:
<img width="399" alt="Screen Shot 2021-05-21 at 5 11 19 AM" src="https://user-images.githubusercontent.com/1994818/119136602-ba148d80-b9f4-11eb-977e-c72892432690.png">

Reply:
>It's worth noting that we're using React Toastify in an unorthodox way. Per its docs, we need to import their styles via import 'react-toastify/dist/ReactToastify.css';, but that never worked, and hence we had to copy over those styles to our repo.

Hm, interesting. Yeah, it'll be good to get PUI integrated soon. I understand react-toastify brings a lot more features with it. What's more, is that PUI's "Snackbar" component is not using react-toastify under the hood as I originally thought. It's a very simple implementation of toasts but covers our needs just fine so far. I'll have to add a bottom position option for the PUI Snackbars once we are using it on the Wallet though.